RadixWallet/Features/Signing/Signer.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+import Sargon
+
+// MARK: - Signer
+// FIXME: move elsewhere. not really part of Profile... but.. where? We need some kind of shared target for higher level models that can depend on Profile. We lack such a package right now.
```

Comment:
Why not under `MIGRATE_TO_SARGON`?

Reply:
not planned to do it "soon", basically every single type which is not a `View`, `Reducer` or a `Client` will be moved into Sargon.